### DISCUSSION TITLE
[Kavita] Show series year (or something) when trying to add CBL

### DISCUSSION THREAD

**Discussion by rad96** [Kareadita/Kavita#3207] [Ideas]
### Idea Description

I have no idea which Year of X-Men I'm missing here and that require me to go and check CBL content itself manually
![изображение](https://github.com/user-attachments/assets/5aa2a61e-3c41-4a19-aaa4-a7eeb313a916)

### Idea Category

User Experience

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
This is a good idea. I haven't been giving CBL much love yet because I'm going to rewrite the whole system with the V2 spec.
